src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,17 +25,20 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryDao countryDao = new CountryDaoImpl(sessionFactory);
```

Comment:
You can do it without creating dao variables

Reply:
What do u mean?